sdk/core/Azure.Core/src/AzureKeyCredential.cs

```diff
@@ -1,28 +1,30 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.ClientModel;
```

Comment:
This one has a small enough implementation surface that if we didn't want to make it inherit, I don't think that would be a problem.

Reply:
Personally, I'm not sure that I see much benefit to inheritance other than consistency with other patterns.   I don't have a strong opinion either way.

Reply:
One advantage I see is that if we inherit, we will be able to use one policy to inject the key into the request.

Reply:
Good point.  We will inherit.